python/src/aiconfig/editor/server/server.py

```diff
@@ -54,26 +120,42 @@ def test():
     return {"key": 6}
 
 
-def run_backend_server(edit_config: EditConfig) -> Result[int, str]:
+@app.route("/api/run")
+async def run():
+    ss = _get_server_state(app)
+    result = await ss.aiconfig_runtime.run(
+        "gen_packing_list",
+        #
+        params={"location": "central park"},
+    )
+    text = result.data[0]
+    return f"<p>{text}</p>"
+
+
+def run_backend_server(edit_config: EditServerConfig) -> Result[int, str]:
     LOGGER.setLevel(edit_config.log_level)
     LOGGER.info("Edit config: %s", edit_config.model_dump_json())
-    LOGGER.info(f"Editor server running on http://localhost:{edit_config.server_port}")
+    LOGGER.info(f"Starting server on http://localhost:{edit_config.server_port}")
 
     app.server_state = ServerState()  # type: ignore
     _init_server_state(app, edit_config)
 
-    if edit_config.server_mode not in {"debug", "prod"}:
+    if edit_config.server_mode not in {"debug_servers", "debug_backend", "prod"}:
         return Err(f"Unknown server mode: {edit_config.server_mode}")
 
-    debug = edit_config.server_mode == "debug"
+    debug = edit_config.server_mode in ["debug_servers", "debug_backend"]
     LOGGER.info(f"Running in {edit_config.server_mode} mode")
-    app.run(port=edit_config.server_port, debug=debug)
+    app.run(port=edit_config.server_port, debug=debug, use_reloader=True)
     return Ok(0)
 
 
-def _init_server_state(app: Flask, edit_config: EditConfig) -> None:
+def _init_server_state(app: Flask, edit_config: EditServerConfig) -> None:
+    if edit_config.parsers_module_path is not None:
+        LOGGER.info(f"Importing parsers module from {edit_config.parsers_module_path}")
+        _load_user_module_from_path_and_register_model_parsers(edit_config.parsers_module_path)
```

Comment:
Would be great to add a route to set this dynamically as well, since a developer can specify this while editing an aiconfig

Reply:
(can be a follow-up diff)

Reply:
I did this later in the stack